handbook/data-stores/mongodb.md

```diff
@@ -0,0 +1,133 @@
+# MongoDB
+
+The majority of SystemLink services use [MongoDB](https://www.mongodb.com) as the primary database. Default installations of SystemLink include MongoDB community Edition that launches when the SystemLink server boots. This is referred to as a [*single node*](#single-node-deployments) deployment.
+
+You can also host MongoDB on a separate stand-alone server, a [replica set](https://docs.mongodb.com/manual/replication/) of 3 or more servers, or use MongoDB's Platform as a service offering: [MongoDB Atlas](https://www.mongodb.com/cloud/atlas). This is referred to as a [*multi node*](#multi-node-deployments) deployment.
```

Comment:
```suggestion
You can also host MongoDB on a separate stand-alone server, a [replica set](https://docs.mongodb.com/manual/replication/) of 3 or more servers, or use MongoDB's Platform as a service offering, [MongoDB Atlas](https://www.mongodb.com/cloud/atlas). This is referred to as a [*multi node*](#multi-node-deployments) deployment.
```

Reply:
Is it intentional that only "Platform" is capitalized in "Platform as a service offering"? I would expect either nothing to be capitalized or more than the first word.